app/src/main/resources/swagger/api-docs.json

```diff
@@ -1503,9 +1502,7 @@
           },
           "digitalAddress" : {
             "type" : "string",
-            "description" : "Digital institution address",
-            "format" : "email",
-            "example" : "email@example.com"
+            "description" : "Digital institution address"
```

Comment:
Insititution digital address

Reply:
I have opened a specific pr for fix swagger documentation